### DISCUSSION TITLE
Link to GitHub repositories on pages

### DISCUSSION THREAD

**Discussion by guylepage3** [vercel/community#6083] [Feature Question]
### Question

As a developer utilizing your example starter templates, having a link from the Vercel template page to the corresponding GitHub repository page would be VERY helpful.

For example a link from https://vercel.com/templates/next.js/subscription-starter to https://github.com/vercel/nextjs-subscription-payments

Thanks

**Comment by amyegan** [comment]
Hey @guylepage3. A link to the repo is available near the top of the template listing, but I can see how this would be easy to miss. 

<img width="466" alt="Screenshot 2024-03-12 at 9 52 58 AM" src="https://github.com/vercel/community/assets/8247105/53d6a9d2-6a5c-4ae4-ae5e-ba7edf043b61">

I'll share this feedback with the team to see if we can make it a little more clear that the repo link is there. Thanks for pointing this out!

**Reply by guylepage3** [reply]
@amyegan Thank you. And yes. Please mention it. I didn't even think that was a link let alone the link to a GitHub repository. Thx again. :)
